install.sh

```diff
@@ -45,8 +45,9 @@ echo
 
 echo -e "\033[35m Usage: $ datree test ~/.datree/k8s-demo.yaml"
 
-echo -e " Using Helm? => https://hub.datree.io/helm-plugin \033[0m"
+echo -e " Using Helm? => https://hub.datree.io/helm-plugin"
 
+echo -e " Run 'datree completion $SHELL' to generate shell autocompletions \033[0m"
```

Comment:
I use zsh as shell and running `echo $SHELL` outputs `/bin/zsh`
using that as an argument in the `completion` comment will not work because it expects `zsh` as value.

I think we can simply add a seperate print for each shell and let the user decide what he wants to do

Reply:
Also, running the command `datree completion $shell` will not result in adding auto completion. 
I suggest you run `datree completion -h` and see what needs to be done to add completion and try it yourself

Reply:
Thanks for pointing this error. I configured autocompletion then I pushed changes but yeah wrote `$SHELL` keyword later on and pushed my changes without trying it. Since it is a echo command it didn't throw any error and I didn't noticed it.

> I think we can simply add a seperate print for each shell and let the user decide what he wants to do

In my opinion, we should just print `datree completion -h` only because adding each separate print will not look that good specially in case of bash i) linux and ii) macos. Moreover `datree completion -h` prints more structured output with some other info as compared to our own echo statements.

What do you think?

Reply:
Yep, I agree, just instruct the user to run "datree completion -h" to view to instructions :)